README.md

```diff
@@ -0,0 +1,23 @@
+# CCAO Data Infrastructure
+
+This repository stores the code for the CCAO Data Department's ETL
+pipelines and data lakehouse. This infrastructure supports the Data Team's
+modeling, reporting, and data integrity work.
+
+## Quick Links
+
+- [:file_folder: dbt Data Catalog](https://ccao-data.github.io/data-architecture/#!/overview) -
+  Documentation for all CCAO data lakehouse tables and views
+- [:nut_and_bolt: dbt README](/dbt/README.md) - How to develop CCAO data
+  infrastructure using dbt
+- [:ballot_box_with_check: dbt Test Documentation](/dbt/tests/generic/README.md) -
```

Comment:
**[Nitpick, non-blocking]** Elsewhere we've been using the test tube emoji (🧪) to represent dbt tests, I don't feel strongly about it but we should be consistent regardless!

```suggestion
- [:test_tube: dbt Test Documentation](/dbt/tests/generic/README.md) -
```